### PR TITLE
ODS-5248 Update Ed-Fi-Extensions to target net6

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

File: Extensions/EdFi.Ods.Extensions.Homograph/EdFi.Ods.Extensions.Homograph.csproj [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
```
Comment (bptillman) [inline Extensions/EdFi.Ods.Extensions.Homograph/EdFi.Ods.Extensions.Homograph.csproj:3]: We should also update Newstonsoft.Json in here to `13.0.1` like what is being done in `EdFi.Ods.Profiles.Sample.csproj` .

Comment (bptillman) [reply Extensions/EdFi.Ods.Extensions.Homograph/EdFi.Ods.Extensions.Homograph.csproj:3]: `Autofac.Extensions.DependencyInjection` was also updated to `7.2.0` as part of the net6 changes in the API so we should update it to the same version here.

File: Extensions/EdFi.Ods.Extensions.Sample/EdFi.Ods.Extensions.Sample.csproj [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
```
Comment (bptillman) [inline Extensions/EdFi.Ods.Extensions.Sample/EdFi.Ods.Extensions.Sample.csproj:3]: Same comment here about updating `Autofac.Extensions.DependencyInjection` to `7.2.0`

File: Extensions/EdFi.Ods.Extensions.SampleStudentTranscript/EdFi.Ods.Extensions.SampleStudentTranscript.csproj [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
```
Comment (bptillman) [inline Extensions/EdFi.Ods.Extensions.SampleStudentTranscript/EdFi.Ods.Extensions.SampleStudentTranscript.csproj:3]: Same comment here about updating `Autofac.Extensions.DependencyInjection` to `7.2.0` and `Newtonsoft.Json` to `13.0.1`

File: Extensions/EdFi.Ods.Extensions.SampleStudentTransportation/EdFi.Ods.Extensions.SampleStudentTransportation.csproj [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
```
Comment (bptillman) [inline Extensions/EdFi.Ods.Extensions.SampleStudentTransportation/EdFi.Ods.Extensions.SampleStudentTransportation.csproj:3]: Same comment here about updating `Autofac.Extensions.DependencyInjection` to `7.2.0`

File: Extensions/EdFi.Ods.Extensions.TPDM/EdFi.Ods.Extensions.TPDM.csproj [hunk 5]
```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
```
Comment (bptillman) [inline Extensions/EdFi.Ods.Extensions.TPDM/EdFi.Ods.Extensions.TPDM.csproj:3]: Same comment here about updating `Autofac.Extensions.DependencyInjection` to `7.2.0` and `Newtonsoft.Json` to `13.0.1`

**Review by bptillman — CHANGES_REQUESTED** [review]
The changes that have been made look good. Just a few additional changes to keep the packages also in line with the updates made to the main repo as well and then this should be good to go.

File: Extensions/EdFi.Ods.Extensions.Sample/EdFi.Ods.Extensions.Sample.csproj (hunk 2 above)
Comment (semalaiappan) [reply Extensions/EdFi.Ods.Extensions.Sample/EdFi.Ods.Extensions.Sample.csproj:3]: updated

File: Extensions/EdFi.Ods.Extensions.SampleStudentTranscript/EdFi.Ods.Extensions.SampleStudentTranscript.csproj (hunk 3 above)
Comment (semalaiappan) [reply Extensions/EdFi.Ods.Extensions.SampleStudentTranscript/EdFi.Ods.Extensions.SampleStudentTranscript.csproj:3]: updated

File: Extensions/EdFi.Ods.Extensions.SampleStudentTransportation/EdFi.Ods.Extensions.SampleStudentTransportation.csproj (hunk 4 above)
Comment (semalaiappan) [reply Extensions/EdFi.Ods.Extensions.SampleStudentTransportation/EdFi.Ods.Extensions.SampleStudentTransportation.csproj:3]: updated

File: Extensions/EdFi.Ods.Extensions.TPDM/EdFi.Ods.Extensions.TPDM.csproj (hunk 5 above)
Comment (semalaiappan) [reply Extensions/EdFi.Ods.Extensions.TPDM/EdFi.Ods.Extensions.TPDM.csproj:3]: updated

File: Extensions/EdFi.Ods.Extensions.Homograph/EdFi.Ods.Extensions.Homograph.csproj (hunk 1 above)
Comment (semalaiappan) [reply Extensions/EdFi.Ods.Extensions.Homograph/EdFi.Ods.Extensions.Homograph.csproj:3]: yes updated

**Review by bptillman — APPROVED** [review]

**Pull request merged**